lambda_uploader/package.py

```diff
@@ -223,6 +224,7 @@ def package(self, ignore=[]):
             those files when creating the zip file. The paths to be matched are
             local to the source root.
         """
+        ignore = ignore or []
```

Comment:
I don't understand why this was added in favor over setting the default for the ignore arg to [].